index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="style.css">
+	<title>MY PORTFOLIO</title>
+</head>
+<body>
+	<section class="hero-section">
+		<nav class="nav-bar">
+			<h1 class="logo">
+				My Logo
+			</h1>
```

Comment:
Kindly make sure your logo is wrapped inside an anchor tag `<a>` and not  header`<h1>`